src/Classes/Cart.php

```diff
@@ -45,5 +47,39 @@ function checkId($id)
         }
     }
 
+    function emailItems()
+    {
+        $db = Database::getinstance();
+        $sql = 'SELECT MAX(order_id) as order_id FROM order_item WHERE user_id = ' . $_SESSION['user_id'] . ' ';
+        $stmt = $db->prepare($sql);
+        $stmt->execute();
+        $result = $stmt->fetch(PDO::FETCH_ASSOC);
+        $order_id = $result['order_id'];
+
+        $sql = 'SELECT * FROM order_item WHERE order_id =' .  $order_id . ' AND user_id =' . $_SESSION['user_id'] . ' ';
+        $stmt = $db->prepare($sql);
+        $stmt->execute();
+        $email_items = $stmt->fetchAll(PDO::FETCH_ASSOC);
+        $sql = 'SELECT * FROM order_complete WHERE order_id =' .  $order_id . ' AND user_id =' . $_SESSION['user_id'] . ' ';
+
+        $stmt = $db->prepare($sql);
+        $stmt->execute();
+        $order_details = $stmt->fetchAll(PDO::FETCH_ASSOC);
+        $i = 1;
+
+        $to = $_SESSION['email'];
```

Comment:
Similar-ish to the above comment, this I believe is the 2nd part of the code-base that now can send an email to the end user. **You don't have to do this**, but think about how making an `Email` class would [DRY ](https://en.wikipedia.org/wiki/Don't_repeat_yourself )up your code and make it easier to send emails in the code base without having to remember which 6 lines to copy/paste each time